apps/nextjs-website/react-components/components/Video/Video.tsx

```diff
@@ -0,0 +1,251 @@
+import { useEffect, useRef, useState } from 'react';
+import { Typography } from '@mui/material';
+import {
+  isJSX,
+  useIsVisible,
+} from '@react-components/types/common/Common.types';
+import {
+  VideoCaptionProps,
+  VideoImageProps,
+  ImageProps,
+  VideoTextProps,
+} from '@react-components/types/Video/Video.types';
+import { TextColor } from '../common/Common.helpers';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import { useTheme } from '@mui/material/styles';
+import Image from 'next/image';
+
+const Video = (props: VideoImageProps) => {
+  const {
+    title,
+    subtitle,
+    caption,
+    src,
+    autoplay = false,
+    loop = false,
+    full = false,
+    theme,
+    fallback,
+    playButtonLabel,
+    isCentered = false,
+    showVideo = true,
+    imagealt,
+    imagesrc,
+  } = props;
+  const videoRef = useRef<HTMLVideoElement>(null);
+  const isVisible = useIsVisible(videoRef);
+  const [error, setError] = useState(false);
+
+  const textColor = TextColor(theme);
+
+  useEffect(() => {
+    if (!isVisible) return;
+    const startVideoWhenVisible = async () => {
+      if (autoplay && isVisible) await play();
+    };
+    startVideoWhenVisible().catch(console.error);
+  }, [isVisible]);
+
+  const play = async (e?: React.MouseEvent) => {
+    try {
+      e?.preventDefault();
+      await videoRef.current?.play();
+    } catch (error) {
+      console.warn(error);
+    }
```

Comment:
```suggestion
    } catch () {}
```
No need to console log errors.